code/src/java/pcgen/cdom/content/UserContent.java

```diff
@@ -109,21 +97,13 @@ public String getExplanation()
 		return explanation;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
-	@SuppressWarnings("PMD.EmptyMethodInAbstractClassShouldBeAbstract")
```

Comment:
Again, losing these is not a good thing.  You are removing effort that has been made to leverage code checking tools and involves lots of time put in to do validation.